### PR TITLE
Setup compathelper and tagbot

### PR DESCRIPTION
**Pull request by lassepe**

### CODE REVIEW

**Comment by lassepe** [comment]
@zsunberg could you review this PR? I can only merge it with an approval.

**Review by zsunberg — APPROVED** [review]
Thanks. Should we change this review requirement setting?

**Comment by lassepe** [comment]
I just realized that I could have used my admin privileges to merge the pr without approval. I guess that would be fine for this kind of uncritical change. For more fundamental changes it would still be good to wait for a review. So I think we can leave the settings as they are right now.

**Pull request merged**
